frontend/quickcheck/src/pages/projectPage/AddMemberButton.jsx

```diff
@@ -32,6 +32,8 @@ export default function AddMemberButton({ onAdd, ...buttonProps }) {
         {...buttonProps}
         aria-label="Add new Member"
         onClick={onOpen}
+        size="lg"
```

Comment:
die parameter hier werden via `buttonProps` in MemberTable zeile 71 übergeben

Reply:
habe die plazierung geändert